iroh/src/client/authors.rs

```diff
@@ -28,4 +31,79 @@ where
         let stream = self.rpc.server_streaming(AuthorListRequest {}).await?;
         Ok(flatten(stream).map_ok(|res| res.author_id))
     }
+
+    /// Export the given author.
+    ///
+    /// Warning: This contains sensitive data.
+    pub async fn export(&self, author: AuthorId) -> Result<Option<Author>> {
+        let res = self.rpc.rpc(AuthorExportRequest { author }).await??;
+        Ok(res.author)
+    }
+
+    /// Import the given author.
+    ///
+    /// Warning: This contains sensitive data.
```

Comment:
remove

Reply:
why? importing contains sensitive data as well

Reply:
Ah, I thought it was a copy-paste error, because I related the `contain` to a return value.
So fine to leave as-is. Or we rephrase the comments to state what and where sensitive data is involved.